app/src/test/java/com/goldenraven/padawanwallet/data/RepositoryTest.kt

```diff
@@ -0,0 +1,130 @@
+package com.goldenraven.padawanwallet.data
```

Comment:
Thanks for adding some real mokito tests! I'm looking into it because I haven't written a lot of those before.